src/main/java/org/jenkinsci/plugins/casc/ObsoleteConfigurationFormat.java

```diff
@@ -0,0 +1,49 @@
+package org.jenkinsci.plugins.casc;
+
+import hudson.Extension;
+import hudson.model.AdministrativeMonitor;
+import org.jenkinsci.plugins.casc.model.CNode;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author <a href="mailto:nicolas.deloof@gmail.com">Nicolas De Loof</a>
+ */
+@Extension
+public class ObsoleteConfigurationFormat extends AdministrativeMonitor {
+
+    private final List<Error> errors = new ArrayList<>();
```

Comment:
This class is not thread-safe, which may be a real problem since `getErrors()` is called from UI. Use R/W locks, CopyOnWrite lists or so